src/components/compress/custom-els/MultiPanel/index.ts

```diff
@@ -1,19 +1,75 @@
-import './styles.css';
+import * as style from './styles.css';
+import { transitionHeight } from '../../../../lib/util';
 
-function getClosestHeading(el: Element) {
-  const closestEl = el.closest('multi-panel > *');
-  if (closestEl && closestEl.classList.contains('panel-heading')) {
-    return closestEl;
+interface CloseAllOptions {
+  exceptFirst?: boolean;
+}
+
+const openOneOnlyAttr = 'open-one-only';
+
+function getClosestHeading(el: Element): HTMLElement | undefined {
+  // Look for the child of multi-panel, but stop at interactive elements like links & buttons
+  const closestEl = el.closest('multi-panel > *, a, button');
```

Comment:
This prevents the panel toggling when you click things like the download button.